tests/integration/test_sns.py

```
@@ -72,3 +74,14 @@ def test_attribute_raw_subscribe(self):
         msgs = sqs_client.receive_message(QueueUrl=queue_url)
         msg_received = msgs['Messages'][0]
         self.assertEqual(message, msg_received['Body'])
+
+    def test_unknown_topic_publish(self):
+        sns_client = aws_stack.connect_to_service('sns')
+        fake_arn = 'arn:aws:sns:us-east-1:123456789012:i_dont_exist'
+        message = u'This is a test message'
+        try:
+            sns_client.publish(TopicArn=fake_arn, Message=message)
+        except ClientError as e:
```

We should raise an assertion error if the previous call on line 83 passes.